pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -110,19 +110,15 @@ func (ssc *defaultStatefulSetControl) performUpdate(
 	if err != nil {
 		return currentRevision, updateRevision, currentStatus, err
 	}
-	klog.V(4).Infof("StatefulSet %s/%s pod status replicas=%d ready=%d current=%d updated=%d",
-		set.Namespace,
-		set.Name,
-		currentStatus.Replicas,
-		currentStatus.ReadyReplicas,
-		currentStatus.CurrentReplicas,
-		currentStatus.UpdatedReplicas)
-
-	klog.V(4).Infof("StatefulSet %s/%s revisions current=%s update=%s",
-		set.Namespace,
-		set.Name,
-		currentStatus.CurrentRevision,
-		currentStatus.UpdateRevision)
+	klog.V(4).InfoS("StatefulSet pod status", "statefulSet", klog.KObj(set),
+		"replicas", currentStatus.Replicas,
+		"ready", currentStatus.ReadyReplicas,
+		"current", currentStatus.CurrentReplicas,
+		"updated", currentStatus.UpdatedReplicas)
+
+	klog.V(4).InfoS("StatefulSet revisions", "statefulSet", klog.KObj(set),
+		"current", currentStatus.CurrentRevision,
```

Comment:
```suggestion
		"currentRevision", currentStatus.CurrentRevision,
```

Reply:
Please make sure usage of key `current` doesn't clash between log calls. Here it is used for revision number, on line 116 is used for replica count. Logs using same key with different intention/value type should be avoided as it could lead log consumer to wrongly interpret them.